packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/FLTGoogleMapJSONConversions.h

```diff
@@ -24,6 +24,9 @@ NS_ASSUME_NONNULL_BEGIN
 + (GMSCoordinateBounds *)coordinateBoundsFromLatLongs:(NSArray *)latlongs;
 + (GMSMapViewType)mapViewTypeFromTypeValue:(NSNumber *)value;
 + (nullable GMSCameraUpdate *)cameraUpdateFromChannelValue:(NSArray *)channelValue;
++ (NSArray<GMSStrokeStyle *> *)strokeStylesFromPatterns:(NSArray *)patterns
+                                            strokeColor:(UIColor *)strokeColor;
++ (NSArray<NSNumber *> *)spanLengthsFromPatterns:(NSArray *)patterns;
```

Comment:
These need declaration comments, per the Obj-C style guide. The existing methods violate the guide, which is problematic, but that shouldn't be perpetuated in new methods.

Reply:
This has not been addressed.